images/win/scripts/Installers/Configure-DynamicPort.ps1

```diff
@@ -0,0 +1,9 @@
+# https://support.microsoft.com/en-us/help/929851/the-default-dynamic-port-range-for-tcp-ip-has-changed-in-windows-vista
+# The new default start port is 49152, and the new default end port is 65535.

```

Comment:
nit: can we add some context in a comment here and say that the ports were changed by VS Enterprise install?